mux/src/tab.rs

```diff
@@ -698,8 +698,8 @@ impl Tab {
         self.inner.lock().get_active_idx()
     }
 
-    pub fn set_active_pane(&self, pane: &Arc<dyn Pane>) {
-        self.inner.lock().set_active_pane(pane)
+    pub fn set_active_pane(&self, pane: &Arc<dyn Pane>, notify: bool) {
```

Comment:
Please switch these `notify: bool` parameters to an enum, as booleans are hard to intuit the meaning of in the code, and are problematic if refactoring later adds additional boolean parameters; it can be tricky to get the ordering right.  Using enums solves both of these problems.

I'm not fantastic at naming, so if you come up better names for both the enum and its variants, I'm all ears!

```rust
#[derive(Debug, Clone, Copy, Eq, PartialEq)]
pub enum FocusNotify {
   Yes,
   No
}
```

so that the call sites look like:

```
tab.set_active_pane(&pane, FocusNotify::No);
```

```suggestion
    pub fn set_active_pane(&self, pane: &Arc<dyn Pane>, notify: FocusNotify) {
```